features/bootstrap/autoloader/autoload.php

```diff
@@ -3,3 +3,27 @@
     $classname = __DIR__ . '/../src/' . str_replace("\\", "/", trim($classname, "\\")) . ".php";
     if (file_exists($classname)) { include $classname; }
 });
+if (!class_exists('FakeLoader')) {
```

Comment:
Having this class created in the bootstrap makes the behaviour in the gherkin scenarios harder to understand.

Can it please be generated based on a Given in those steps? 

Currently to understand the behaviour in a particular Scenario I have to refer to this file

Reply:
I think I could, but you would still have to refer to this file because of the `return FakeLoader();` statement that still would have to be here.

Reply:
I think that would be ok - the key is to get the 'why' into the scenario